src/components/reference/_cli-help-scan-output.md

```diff
@@ -163,7 +163,7 @@ OPTIONS
            have time limit. Defaults to 0 s for all CLI scans. For CI scans,
            it defaults to 3 hours.
 
-       -j VAL, --jobs=VAL (absent=4)
+       -j VAL, --jobs=VAL
```

Comment:
I'm not sure if this text is automatically generated based on the output of running `semgrep --help`.

Reply:
@adamcohen2 Yes, you're right, it's automatically generated from the main semgrep repo. I'll let the team know.